pandas/tests/internals/test_internals.py

```diff
@@ -1249,15 +1248,16 @@ class TestCanHoldElement(object):
         operator.pow,
     ], ids=lambda x: x.__name__)
     def test_binop_other(self, op, value, dtype):
-        skip = {(operator.add, 'bool'),
-                (operator.sub, 'bool'),
-                (operator.mul, 'bool'),
-                (operator.truediv, 'bool'),
-                (operator.mod, 'i8'),
-                (operator.mod, 'complex128'),
-                (operator.mod, '<M8[ns]'),
-                (operator.mod, '<m8[ns]'),
-                (operator.pow, 'bool')}
+        skip = {
+            (operator.add, 'bool'),
+            (operator.sub, 'bool'),
+            (operator.mul, 'bool'),
+            (operator.truediv, 'bool'),
+            (operator.mod, 'i8'),
+            (operator.mod, 'complex128'),
+            (operator.mod, '<m8[ns]'),
+            (operator.pow, 'bool'),
+        }
```

Comment:
Pls avoid formatting changes where convenient, makes it harder for reviewers to identify the actual changes.